lvm/lvm-util.c

```diff
@@ -67,7 +67,8 @@ lvm_copy_name(char *dst, const char *src, size_t size)
 	if (strnlen(src, size) == size)
 		return -ENAMETOOLONG;
 
-	strcpy(dst, src);
+	strncpy(dst, src, size);
+	dst[size - 1] = '\0';
```

Comment:
What if size is 0, does the caller guarantee it isn't?

Reply:
Even if it were the check at line 67 would have returned `-ENAMETOOLONG`